python_modules/libraries/dagster-dbt/dagster_dbt/include/dbt_scaffold/definitions.py

```diff
@@ -0,0 +1,32 @@
+from pathlib import Path
+
+from dagster import Definitions
+
+from dagster_dbt import DbtCli, DbtManifest, dbt_assets
+
+DBT_PROJECT_DIR = Path(__file__).parent.joinpath("..", "jaffle_shop")
+DBT_MANIFEST_PATH = DBT_PROJECT_DIR.joinpath("target", "manifest.json")
+DBT_MANIFEST = DbtManifest.read(path=DBT_MANIFEST_PATH)
+
+
+@dbt_assets(manifest=DBT_MANIFEST)
+def my_dbt_assets(dbt: DbtCli):
+    yield from dbt.cli(["build"], manifest=DBT_MANIFEST).stream()
+
+
+schedules = [
```

Comment:
My takeaway from our earlier conversations had been that we would put this in a separate file, so that users don't need to be confronted with all this other intimidating stuff in order to mess with their basic schedules.  It has some downsides in terms of having imports between files, but my intuition is that it's worth it to make things more friendly.

Reply:
Your takeaway is right.

However, I've put everything in one file for this PR to have a baseline, similar to what I did for the `.build_schedule` API (https://github.com/dagster-io/dagster/pull/14992 + https://github.com/dagster-io/dagster/pull/14997).

Like last time, I'll put the changes in folder structure in another PR, so we can visually see the changes in comparison to this single-file option.

Reply:
Got it - sounds good